operator/src/test/java/oracle/kubernetes/operator/create/CreateDomainGeneratedFilesBaseTest.java

```diff
@@ -333,7 +333,6 @@ protected void assertThatActualCreateDomainPyIsCorrect(String actualCreateDomain
         getInputs().getAdminServerName(),
         getInputs().getManagedServerNameBase(),
         getInputs().getDomainUID() + "-" + getInputs().getAdminServerName(),
-        getInputs().getDomainUID() + "-" + getInputs().getManagedServerNameBase(),
```

Comment:
Update the commented out create-domain.py just above this code to remove the machine related lines so that it's obvious why you needed to remove looking for 'getInputs().getDomainUID() + "-" + getInputs().getManagedServerNameBase(),'